utilities/backupable/backupable_db.cc

```diff
@@ -2090,7 +2090,8 @@ Status BackupEngineImpl::AddBackupFileWorkItem(
   if (shared && !same_path) {
     // Should be in shared directory but not a live path, check existence in
     // shared directory
-    Status exist = backup_env_->FileExists(final_dest_path);
+    IOStatus exist =
+        status_to_io_status(backup_env_->FileExists(final_dest_path));
     if (exist.ok()) {
```

Comment:
IOStatus exist = backup_env_->GetFileSystem()->FileExists(...) ?

Reply:
> IOStatus exist = backup_env_->GetFileSystem()->FileExists(...) ?

Actually, I plan to replace most of the backup_env_->(file or directory related call) into FileSystem call in next PR, including backup_env_->CreateDir, DeleteFile, GetChildren, DeleteDir, CreateDirIfMissing, NewDirectory, FileExists